components/Workshop/WorkshopListingInterestModal.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+
+import React from "react";
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import { BsEmojiHeartEyes } from "react-icons/bs";
+
+interface IProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onSubmit: () => void;
+}
+
+export default function WorkshopListingInterestModal({
+  isOpen,
+  onClose,
+  onSubmit,
+}: IProps) {
+  const expressInterestText: string = `You can still express interest in this workshop even if there are no available sessions. We will notify the host and they might schedule new ones.`;
```

Comment:
I would rewrite this a bit. This is just a suggestion, happy for you to play around with it:

Let the host know you’re interested!
When you show interest, we notify the host that there’s demand. That could be just the nudge they need to schedule more sessions.